addon/components/o-s-s/phone-number-input.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="phone-number-container fx-1" ...attributes>
+  <div class="phone-number-input upf-input fx-row fx-1 fx-xalign-center">
+    <div class="country-selector fx-row" role="button" {{on "click" this.toggleCountrySelector}}>
+      <div class="fflag fflag-{{this.selectedCountry.id}} ff-sm ff-wave ff-rounded"></div>
+      {{#if this.countrySelectorShown}}
+        <i class="fal fa-chevron-up margin-left-xxx-sm"></i>
```

Comment:
so we start using the `fal` ?

Reply:
In this case it is just a thinner chevron ; is that an issue ? 🤔